cmd/challenge/catalog.go

```diff
@@ -30,11 +31,18 @@ func newCatalogCommand(cli labcli.CLI) *cobra.Command {
 
 	flags := cmd.Flags()
 
-	flags.StringVar(
+	flags.StringSliceVar(
 		&opts.category,
 		"category",
-		"",
-		`Category to filter by - one of linux, containers, kubernetes, ... (an empty string means all)`,
+		[]string{},
```

Comment:
The `an empty string means all` part is probably a leftover from the previous implementation.

Reply:
Done